src/css-grids/css-grids.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CSS-Grids</title>
+    <link rel="stylesheet" href="css-grids-styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower&display=swap" rel="stylesheet"> 
+
+</head>
+<body>
+    <main class="main-container">
+        <div class="image">
+            <img class="kittens" src="./images/kittens2.jfif" alt="Pictures of kittens"></img>
+        </div>
+        <nav class="navbar">
+            <img img-style="float: left;" class="downsize" src="https://cdn.evbuc.com/eventlogos/188084824/acastandardcirclefullname.png" alt="Austin coding Academy Logo">
+            <a class="navbar1" href="">Login</a>
+            <a class="navbar1" href="">Stuff</a>
+            <a class="navbar1" href="https://cssbattle.dev/">CSS Battle</a>
+            <a class="navbar1" href="">Login</a>
+        </nav>
+        <div class="todo">To do List:<ul>
+        <li>Get CSS-Grids & Flex down</li>
+        <li>More pictures of kittens!!!!</li>
+        <li>Add full Media Queries to Check Point</li>
+        <li>World Domination</li>
+        <li class="proof">Prove that Comic-Sans is cool!</li>
+    </div>
+    <div class="button-content">
+        <div class="button-row">
+          <div class="button-element">mc</div>
+          <div class="button-element">mc2</div>
+          <div class="button-element">/</div>
+          <div class="button-element">X</div>
+        </div>
+        <div class="button-row">
+          <div class="class button-element">7</div>
+          <div class="class button-element">8</div>
+          <div class="class button-element">9</div>
+          <div class="class button-element">-</div>
+        </div>
+        <div class="button-row">
+          <div class="class button-element">4</div>
+          <div class="class button-element">5</div>
+          <div class="class button-element">6</div>
+          <div class="class button-element">=</div>
+        </div>
+        <div class="button-row">
+          <div class="class button-element">1</div>
+          <div class="class button-element">2</div>
+          <div class="class button-element">3</div>
+          <div class="class button-element">0</div>
+      </div>
+      <style style="text/css">
+        .scroll-up {
```

Comment:
i'm assuming this is something other than the grid styling. put it in it's own css stylesheet and add the link to the head element on this page

Reply:
I just moved it to the primary css file and it clear alot of clutter in the HTML